imaging-hub.data-commons.org/manifests/hatchery/hatchery.json

```diff
@@ -0,0 +1,222 @@
+{
+  "user-namespace": "jupyter-pods",
+  "prisma": {
+    "enable": true,
+    "console-address": "https://us-west1.cloud.twistlock.com/gov-3229443"
+  },
+  "sub-dir": "/lw-workspace",
+  "user-volume-size": "10Gi",
+  "default-pay-model": {
+    "workspace_type": "Trial Workspace",
+    "local": true
+  },
+  "pay-models-dynamodb-table": "bmh-admin-portal-backend-bmhworkspacetableD4D73E43-34AAJOR7A5AG",
+  "sidecar": {
+    "cpu-limit": "1.0",
+    "memory-limit": "256Mi",
+    "image": "quay.io/cdis/ecs-ws-sidecar:2024.02",
+    "env": {
+      "NAMESPACE": "default",
+      "HOSTNAME": "brh.data-commons.org"
+    },
+    "args": [],
+    "command": [
+      "/bin/bash",
+      "./sidecar.sh"
+    ],
+    "lifecycle-pre-stop": [
+      "su",
+      "-c",
+      "echo test",
+      "-s",
+      "/bin/sh",
+      "root"
+    ]
+  },
+  "nextflow-global": {
+    "sample-config-public-image": "public.ecr.aws/u5x5h6w3/nextflow-approved/public:gen3-amazonlinux-base-AL2023fix",
+    "imagebuilder-reader-role-arn": "arn:aws:iam::143731057154:role/nextflow-imagebuilder-reader"
+  },
+  "containers": [
```

Comment:
let's just set an empty list of containers? i don't think we'll be using the workspace in BIH yet, and these options are too specific to BRH

Reply:
I think I'll remove hatchery configs for now.